generic3g/specs/VariableSpec.F90

```diff
@@ -192,7 +199,7 @@ function make_FieldSpec(this, geom, rc) result(field_spec)
 
       units = get_units(this, _RC)
 
-      field_spec = new_FieldSpec_geom(geom=geom, typekind=this%typekind, ungridded_dims=this%ungridded_dims, &
+      field_spec = new_FieldSpec_geom(geom=geom, vertical_geom = vertical_geom, vertical_dim = this%vertical_dim_spec, typekind=this%typekind, ungridded_dims=this%ungridded_dims, &
```

Comment:
I'm hoping that this procedure is not currently used, as `geom` is not known at the time of a FieldSpec being created.   Instead it is passed in at the allocate step.  Look at `allocate*()` in `./registry/HierarchicalRegistry.F90`

Reply:
OK - this one was my poor memory.  The "geom" is available at this stage, as we only create the VarSpec during SetServices.